src/altinn-app-frontend/src/utils/appUrlHelper.test.ts

```diff
@@ -1,163 +1,226 @@
 import {
+  dataElementUrl,
+  fileTagUrl,
+  fileUploadUrl,
   getCalculatePageOrderUrl,
+  getCreateInstancesUrl,
+  getDataValidationUrl,
   getEnvironmentLoginUrl,
   getFetchFormDynamicsUrl,
   getHostname,
   getLayoutSettingsUrl,
   getLayoutsUrl,
   getOptionsUrl,
+  getProcessNextUrl,
+  getProcessStateUrl,
+  getRedirectUrl,
   getRulehandlerUrl,
   getStatelessFormDataUrl,
+  getUpgradeAuthLevelUrl,
+  getValidationUrl,
+  redirectToUpgrade,
+  textResourcesUrl,
+  updateCookieUrl,
+  validPartiesUrl,
 } from 'src/utils/appUrlHelper';
 
 describe('Frontend urlHelper.ts', () => {
-  describe('getEnvironmentLoginUrl', () => {
-    test('should return correct url when oidc provider is "idporten" and host has 3 subdomains', () => {
-      const oldWindowLocation = window.location;
-      delete window.location;
-      window.location = {
-        ...oldWindowLocation,
-        origin: 'https://ttd.apps.altinn.no',
-        hash: '#/datamodelling',
-        pathname: '/ttd/jesttest/',
-        host: 'https://ttd.apps.altinn.no',
-        href: 'https://ttd.apps.altinn.no/ttd/test',
-      };
-      const oidcProvider = 'idporten';
-      expect(getEnvironmentLoginUrl(oidcProvider)).toContain(
-        'https://platform.altinn.no/authentication/api/v1/authentication?goto=https%3A%2F%2Fttd.apps.altinn.no%2Fttd%2Ftest&iss=idporten',
+  window['instanceId'] = 'instance/id';
```

Comment:
I find this a little confusing, its challenging to see if this is part of the base url or if its a variable with `instance/id`. Would prefer to use something explicit to make it more obvious that this is a variable part of the url.